edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -59,6 +59,10 @@ def get_jwt_claim_mergeable_attributes(self):
         return get_setting('JWT_PAYLOAD_MERGEABLE_USER_ATTRIBUTES')
 
     def authenticate(self, request):
+        # latest version of drf-jwt does not work with other auth header.
+        if request.environ.get('HTTP_AUTHORIZATION', '').startswith('Bear'):
```

Comment:
@robrap https://github.com/Styria-Digital/django-rest-framework-jwt/pull/27/files#diff-bc0105dbb10d8b2bcacf8d191d5d0195R105
This line throws error. Previously it return None.

https://github.com/Styria-Digital/django-rest-framework-jwt/issues/57

Reply:
Would it makes sense to test for not JWT instead?